clean.cmd

```diff
@@ -61,9 +61,9 @@ shift /1
 goto Loop
 
 :Begin
-
-echo Running init-tools.cmd
-call %~dp0init-tools.cmd
+REM If VBCSCompiler.exe is running we need to kill it
+tasklist /fi "imagename eq VBCSCompiler.exe" |find ":" > nul
+if errorlevel 1 taskkill /f /im "VBCSCompiler.exe"
```

Comment:
I'm not sure that it's necessary to kill this process whenever clean.cmd is run, because my understanding is that it only interferes with deleting the Tools directory.  What do you think about only killing the process when the script is deleting the Tools directory (i.e. when either the /t or /all option is used)?  Alternatively, perhaps we could add a new flag for killing that process.